modules/lofreq/1.0/src/bash/lofreq_filter.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script will filter out the non-standard chromosomes and non-ACTG characters from
+# vcf files produced by lofreq. It accepts .qz-compressed files.
+# Usage: 
+#   lofreq_filter.sh input.vcf.gz | gzip > output.vcf.gz
+
+INPUT_FILE="$1"
+
+zcat "${INPUT_FILE}" \
+	| awk '($4=="A" || $4 == "C" || $4=="T" || $4=="G" || /\#/)' \
+	| grep -E -v "v1|v2|GL0"
```

Comment:
This is a bit hacky. Shouldn't we really be checking for exact match of the first column to one of the chromosomes in the main chromosome list?

Reply:
Sound like a time for me to learn details of regex in awk. I changed it to the following `awk -e '$1 ~ /^#|^(chr)*[1-9]$|^(chr)*[1-2][0-9]$|^(chr)*X$/ {print$0}'` and tested on both grch37 and hg38 files. Currently running demo project to make sure it is working in the whole run.

Reply:
Nice!
If I understand correctly, this prints any header line and any line in which the first column matches the pattern chr (optional) plus numeric or X. Dropping chrY is intentional, right?

Reply:
Notably, it's a lot more succinct if we do this with Perl:
```
perl -ne 'print if /^#/ || /^(chr)*\d+\s.+/'
```